src/prefect/deployments.py

```diff
@@ -167,12 +168,24 @@ async def load_flow_from_flow_run(
             storage_block = Block._from_block_document(storage_document)
         else:
             basepath = deployment.path or Path(deployment.manifest_path).parent
+            # Replace {workflows_dir} in the deployment path with the local workflow storage path
+            basepath = str(basepath).format(
+                workflows_dir=PREFECT_WORKER_WORKFLOW_STORAGE_PATH.value()
+            )
             storage_block = LocalFileSystem(basepath=basepath)
 
         sys.path.insert(0, ".")
 
-        logger.info(f"Downloading flow code from storage at {deployment.path!r}")
-        await storage_block.get_directory(from_path=deployment.path, local_path=".")
+        # Replace {workflows_dir} in the deployment path with the local workflow storage path
```

Comment:
This variable feels a little awkward to me, it doesn't match the setting closely and seems hard to remember them both. Is there another pattern we can use? Perhaps it's a `PREFECT_WORKER_FLOW_BASEPATH` and we prepend it to all local storage paths if the variable is set?

Reply:
Yea I think you're onto something - I think we could allow the variable to be referenced explicitly within the path